Public/Src/Cache/ContentStore/Interfaces/Results/ResultBase.cs

```diff
@@ -111,6 +111,11 @@ protected ResultBase(ResultBase other, string message)
         /// </summary>
         public TimeSpan Duration { get; set; }
 
+        /// <summary>
+        /// Gets or sets additional information that can be used for tracing purposes.
+        /// </summary>
+        public string ExtraDiagnosticsMessage { get; set; }

```

Comment:
>ExtraDiagnosticsMessage [](start = 22, length = 23)

`Extra` tells us nothing about the property. Would it be helpful to rename it for its actual purpose (e.g. `LockDurationMs`)?

Reply:
I think it makes sense to have it be more general since we can use it for other purposes. Also having lock duration in ResultBase is a bit weird.

---
In reply to: [280252679](https://github.com/Microsoft/BuildXL/pull/233#discussion_r280252679) [](ancestors = 280252679)

Reply:
Repurposed Diagnostics field for this.

---
In reply to: [280482874](https://github.com/Microsoft/BuildXL/pull/233#discussion_r280482874) [](ancestors = 280482874,280252679)